recipes-kernel/linux/linux-nilrt_git.bb

```diff
@@ -5,6 +5,10 @@ LINUX_VERSION:xilinx-zynq = "4.14"
 
 require linux-nilrt.inc
 
+SRC_URI += "\
```

Comment:
Seeing that `extra.cfg` is x64 specific and we'll have ARM building from same codebase soon, I think we should make it `extra_x64.cfg` or something like that and make this a `SRC_URI:append:x64`.

What do you think @gratian ?

Reply:
@chaitu236 Yes, It think that makes sense.